runtime/plugin/nvim/spellfile.lua

```diff
@@ -1,16 +1,12 @@
-vim.g.loaded_spellfile_plugin = true
-
---- Downloads missing .spl file.
----
---- @param args { bufnr: integer, match: string }
-local function on_spellfile_missing(args)
-  local spellfile = require('nvim.spellfile')
-  spellfile.load_file(args.match)
+if vim.g.loaded_spellfile_plugin ~= nil then
+  return
 end
+vim.g.loaded_spellfile_plugin = true
 
 vim.api.nvim_create_autocmd('SpellFileMissing', {
-  group = vim.api.nvim_create_augroup('nvim_spellfile', { clear = true }),
-  pattern = '*',
+  group = vim.api.nvim_create_augroup('nvim_spellfile', {}),
```

Comment:
not sure how i missed this. we really need custom lint rules!


```suggestion
  group = vim.api.nvim_create_augroup('nvim.spellfile', {}),
```